charts/router-mongo/templates/service.yaml

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
I wonder if this will cause issues with mongo's replica set logic. Iirc, the hostname that is defined in the replica set config has to match the host set on the mongo host. It should be possible to work around it though

Reply:
Yeah it'll only work for one replica, for sure. Here's my [rollout plan](https://docs.google.com/document/d/1_6nAsqAnrYpCOMCVw1cde6xdymgnRPCv6bvbQZ_Yyic/edit#) so far.

Reply:
Figured I should stick a description on it — thanks!